sql/src/main/java/io/crate/jobs/PageDownstreamContext.java

```diff
@@ -59,6 +61,7 @@
     public PageDownstreamContext(int id,
                                  String name,
                                  PageDownstream pageDownstream,
+                                 RowReceiver pageDownstreamRowReceiver,
```

Comment:
prepare() is not called when the rowReceiver is attached to a projectorChain like in this case